index.html

```diff
@@ -0,0 +1,168 @@
+      <span class="schedule-title">Schedule</span>
+      <hr class="solid" />
+      <div class="schedule-container">
+        <article class="schedule-card">
+          <i class="fa fa-ticket"></i>
+          <h3 class="card-title">Ticket Registration</h3>
+          <p class="card-description">
+            For this edition from March 5th to 27th, 5,000 tickets are on
+            sale. The Puple coders seminar will have some of the biggest 
+            names in the industry sharing there ideas and will be taking 
+            Questions from the audience on tecnical issues.All are welcome
+          </p>
+        </article>
+        <article class="schedule-card">
+          <i class="fa fa-torii-gate"></i>
+          <h3 class="card-title">Gates Open</h3>
+          <p class="card-description">
+            The gate opens for all participants at 10:00 am on the Friday 5th
+            May, and is open 10am to 4pm during the event hours a day throughout
+             the event.Timing is important and we will kindly request all our 
+             guests to be settled 30 minutes before the starting time.Thank you in advance.
+          </p>
+        </article>
+        <article class="schedule-card">
+          <i class="fa fa-moon"></i>
+          <h3 class="card-title">Evening Sessions</h3>
+          <p class="card-description">
+            There will be food and drinks served for all our guests from 4:30 pm to 10pm
+             to allow all our guests to share there learning experience and share 
+             technicals issues learned during the session 
+          </p>
+        </article>
+        <article class="schedule-card">
+          <i class="fa fa-volume-up"></i>
+          <h3 class="card-title">Bulletin</h3>
+          <p class="card-description">
+            All guest who are not settled in 30 mins before the session will not be allowed to enter.
+             Food and drinks will be served for free within the complex to avoid movement a 
+             link will be shared where you can choose your prefered meal from our menu.
+             Any special request will be charged extra. Thank you in advance
+          </p>
+        </article>
+      </div>
+      <a href="" class="whole-schedule">SEE THE WHOLE SCHEDULE</a>
+      <button class="btn btn-schedule">Get Your Ticket</button>
+    </section>
+<!-- 
+    =============== Featured Speakers=============== -->
+    <div id="featureActs" class="featured-acts">
+      <span class="featured-title">Featured Speakers</span>
+      <hr class="featured-solid" />
+      <div id="actContainer" class="acts-container"></div>
+      <button type="button" id="collapse" class="btn btn-collapse">
+        <span id="expand">More<i class="fas fa-angle-down"></i></span>
+        <span id="exit" class="hide">Less<i class="fas fa-angle-up"></i></span>
+      </button>
+    </div> 
+<!-- ==============FOOTER SECTION=============== -->
+    <div class="partner-section">
```

Comment:
- [ ] Consider using a section element to wrap the partners section rather than a div element. This is more semantically correct.